.php-cs-fixer.dist.php

```diff
@@ -66,3 +66,11 @@
     )
     ->setCacheFile('.php-cs-fixer.cache')
 ;
+
+# PHP-CS-Fixer parallel runner available since v3.57.0, let's enable CPU auto-detection when available.
+# @see https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7777
+if ($config instanceof PhpCsFixer\ParallelAwareConfigInterface) {
```

Comment:
I don't think, we need this safeguard here. The config is regularly updated under the assumption that the latest version of CS Fixer is used. I someone used an outdated version, they'd get unexpected changes or errors about unknown rules anyway.

Reply:
Not having the condition would work for me as well. fabbot is ready 🟢

Reply:
Fabbot is ready, but people who work with the project can have PHARs pre-3.57 (not many new rules added since then, if any), so it may fail without this. I added it to be more backward-compatible, but I can remove it if you prefer a hardcoded parallel setup 😊. Just let me know, I'll do this later (currently AFK).

Reply:
I think we can remove it.